### PR TITLE
chore: update nan for node12

### PR DESCRIPTION
**Pull request by deepak1556**

### CODE REVIEW

**Comment by bpasero** [comment]
@sbatten can you take a look

File: package.json [hunk 1]
```diff
@@ -2,7 +2,6 @@
   "name": "vscode-windows-registry",
   "version": "1.0.1",
   "description": "A native node module for accessing the windows registry",
-  "os": ["win32"],
```
Comment (sbatten) [inline package.json]: why do we remove the win32 bit?

Comment (deepak1556) [reply package.json]: Oops mistake, will revert that.

**Comment by deepak1556** [comment]
@sbatten PTAL, thanks!

**Pull request merged**